include/oneapi/dpl/experimental/kt/esimd_radix_sort_onesweep.h

```diff
@@ -581,16 +581,22 @@ onesweep(sycl::queue __q, _Range&& __rng, ::std::size_t __n)
     constexpr uint32_t GLOBAL_OFFSET_SIZE = BINCOUNT * STAGES * sizeof(global_hist_t);
     size_t temp_buffer_size = GLOBAL_OFFSET_SIZE + SYNC_BUFFER_SIZE;
 
-    uint8_t *tmp_buffer = sycl::malloc_device<uint8_t>(temp_buffer_size, __q);
-    auto p_global_offset = reinterpret_cast<uint32_t*>(tmp_buffer);
+    const size_t full_buffer_size_part1 = temp_buffer_size * sizeof(uint8_t);
```

Comment:
Would you mind about removing this line?

Reply:
fixed

Reply:
I prefer to have it